content/api/suppression-list.apib

```diff
@@ -110,25 +146,20 @@ Please note that in the unlikely scenario where your receive a HTTP 5xx level er
                 }
         }
 
-## Search [/suppression-list{?to,from,domain,cursor,limit,per_page,page,sources,types,description}]
-
-### Search for List Entries [GET]
+### Search for List Entries [GET /suppression-list{?to,from,domain,cursor,limit,per_page,page,sources,types,description}]
```

Comment:
Lets move this request to be second to last, right before the summary end point.